devices.js

```diff
@@ -9093,11 +9120,31 @@ const devices = [
     },
     {
         zigbeeModel: ['vivi ZLight'],
-        model: 'LifeControl_RGB_Led',
+        model: 'MCLH-02',
```

Comment:
Make sure to rebase this PR, these changes shouldn't be included here.

Reply:
ok